cpp/arcticdb/version/local_versioned_engine.cpp

```diff
@@ -923,6 +923,15 @@ folly::Future<folly::Unit> delete_trees_responsibly(
 
     folly::Future<folly::Unit> remove_keys_fut;
     if (!dry_run) {
+        for (const auto& key : vks_column_stats) {
```

Comment:
Probably better to compile these out for release builds, eg use `ARCTICDB_TRACE` macro?